srcalc/src/main/webapp/WEB-INF/views/tiles/enterVariables.jsp

```diff
@@ -119,6 +125,10 @@
 	    			"aTargets": [0]
 	    		}]
 	    	});
+        	// Use the CPT code for the proper radio button and set it to checked
+        	$("input:radio[value='${selectedProcedure}']").attr("checked",true);
```

Comment:
I just learned this today, but you should be using prop() instead of attr() here. See this doc: http://api.jquery.com/prop/